src/v2i-hub/SpatPlugin/src/NTCIP1202OIDs.h

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (C) 2024 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#pragma once
+
+#include <string>
+
+namespace NTCIP1202V3{
+    /**
+     * @brief OID for ENABLE_SPAT
+     */
+    static const std::string ENABLE_SPAT_OID = "1.3.6.1.4.1.1206.3.5.2.9.44.1.0";
+    /**
+     * @brief OID for Intersection ID.
+     */
+    static const std::string INTERSECTION_ID = " 1.3.6.1.4.1.1206.4.2.1.17.1.2.1.2";
```

Comment:
Do we have reference to these two OIDs?

Reply:
What kind of reference are you looking for?

Reply:
Where are these OIDs from and what do they mean in an official document?

Reply:
Only the intersection ID. The enable SPAT OID is from before SNMP V3 was part of the standard. I think for security concerns they did not document certain object in the public facing documentation and left all OIDs out of any NTCIP 1202 documentation prior to V3. Here is the V3 guide (https://www.ntcip.org/file/2019/07/NTCIP-1202v0328A.pdf). Here is the V2 guide (https://www.ntcip.org/wp-content/uploads/2018/11/NTCIP1202v0219f.pdf) (V2 is missing OIDs for the reason mentioned above and several objects)